modules/standard/BigInteger.chpl

```diff
@@ -528,12 +528,17 @@ module BigInteger {
     @deprecated("get_str is deprecated - please use a cast to a string or IO methods to get the string representation")
     proc get_str(base: int = 10): string do return this.getStr(base);
 
-    /* Writes this number to a :type:`~IO.fileWriter` */
+    @chpldoc.nodoc
     proc writeThis(writer) throws {
       var s: string;
       s = this.getStr();
       writer.write(s);
     }
+
+    /* Writes this number to a :type:`~IO.fileWriter` */
+    proc serialize(writer, ref serializer) throws {
```

Comment:
I was wondering the same, actually. I think part of the normal documentation would be indicating which type-kind the type will print as (i.e. List/Map/etc).

It would be a bit tedious to repeatedly clarify the point about being invoked by a fileWriter, so I wonder if we could instead document this on the type itself and no-doc the serialize/deserialize methods? Do you think this could wait until doc freeze?